src/snowflake/connector/connection.py

```diff
@@ -3,6 +3,7 @@
 #
 # Copyright (c) 2012-2019 Snowflake Computing Inc. All right reserved.
```

Comment:
Can you update this header?

Reply:
I'm always confused about these, are they supposed to start at the current year. Or what should I have written here?

Reply:
Oh I guess it should be 2020, not 2019. Let me change that!

Reply:
I have also filed SNOW-194973 will do this separately from this PR tho.